gtfs-realtime/spec/en/reference.md

```diff
@@ -251,6 +251,27 @@ The degree of passenger occupancy for the vehicle.
 | _**FULL**_ | _The vehicle is considered full by most measures, but may still be allowing passengers to board._ |
 | _**NOT_ACCEPTING_PASSENGERS**_ | _The vehicle can not accept passengers._ |
```

Comment:
Please remove `NOT_ACCEPTING_PASSENGERS` in the docs and mark it as deprecated in the .proto file.